.gitlab-ci.yml

```diff
@@ -30,7 +30,8 @@ variables:
   RUSTY_CACHIER_COMPRESSION_METHOD: zstd
   NEXTEST_FAILURE_OUTPUT: immediate-final
   NEXTEST_SUCCESS_OUTPUT: final
-  ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.69"
+  # ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.69"
+  ZOMBIENET_IMAGE: "docker.io/paritypr/zombienet:2df6a12b"
```

Comment:
It'd be better not to use docker images owned by `paritypr` here. What's the blocker on using an officially released `paritytech/zombienet` version?

Reply:
Thanks for spotting this.

I needed to fix zombienet: https://github.com/paritytech/zombienet/pull/1406, to make all tests green.
Not sure if it was already released (@pepoviola?)

Reply:
I'll fix it once https://github.com/paritytech/polkadot-sdk/pull/1931 is merged.